pandas/conftest.py

```diff
@@ -89,6 +89,14 @@ def join_type(request):
     return request.param
 
 
+@pytest.fixture(params=[None, np.nan, pd.NaT])
```

Comment:
This is a good idea.  Some more for the list: `np.timedelta64('NaT'), np.datetime64('NaT'), float('nan'), np.float('NaN')`.

Note that `np.float('NaN')` does not return `np.nan` (i.e. does not behave like `pd.NaT`), so this would catch any occurrences in pandas of e.g `if foo is np.nan`